docker-sortinghat/README.md

```diff
@@ -0,0 +1,65 @@
+# SortingHat OpenInfra Docker Image
```

Comment:
I wouldn't say this is a image for OpenInfra. It is the Bitergia Analytics image for SortingHat. This image includes the openifra importer.